internal/ssh/helpers.go

```diff
@@ -0,0 +1,130 @@
+package ssh
+
+import (
+	"errors"
+	"net"
+	"regexp"
+
+	"github.com/infamousjoeg/cyberark-aam-pkiaas/internal/types"
+)
+
```

Comment:
Could we test the Validate functions?

Reply:
I'm going to start the unit tests for the SSH capabilities later this week, but yes :)